scripts/main.js

```diff
@@ -6,11 +6,278 @@ function coolMenu() {
+
+  const Microverse = Object.create(projects);
+
+  Microverse.name = 'Microverse';
+  Microverse.description = 'Learn to code';
+  Microverse.featuredImage.imgSrc = 'resources/Microverse.png';
+  Microverse.featuredImage.imgAlt = 'Microverse Logo';
+  Microverse.technologies.tech1 = 'HTML/CSS';
+  Microverse.technologies.tech2 = 'JavaScript';
+  Microverse.technologies.tech3 = 'C/C++';
+  Microverse.liveVersion = 'https://fdezcaminero.github.io/figmaPortfolio2023/';
+  Microverse.linkToSource = 'https://github.com/fdezcaminero/figmaPortfolio2023';
+
+  document.getElementById('ProjectName5').innerHTML = Microverse.name;
+  document.getElementById('5tech1').innerHTML = Microverse.technologies.tech1;
+  document.getElementById('5tech2').innerHTML = Microverse.technologies.tech2;
+  document.getElementById('5tech3').innerHTML = Microverse.technologies.tech3;
+
+  const YouTube = Object.create(projects);
+
+  YouTube.name = 'YouTube';
+  YouTube.description = 'You Tube';
+  YouTube.featuredImage.imgSrc = 'resources/Youtube.png';
+  YouTube.featuredImage.imgAlt = 'YouTube Logo';
+  YouTube.technologies.tech1 = 'HTML/CSS';
+  YouTube.technologies.tech2 = 'Go';
+  YouTube.technologies.tech3 = 'C/C++';
+  YouTube.liveVersion = 'https://fdezcaminero.github.io/figmaPortfolio2023/';
+  YouTube.linkToSource = 'https://github.com/fdezcaminero/figmaPortfolio2023';
+
+  document.getElementById('ProjectName6').innerHTML = YouTube.name;
+  document.getElementById('6tech1').innerHTML = YouTube.technologies.tech1;
+  document.getElementById('6tech2').innerHTML = YouTube.technologies.tech2;
+  document.getElementById('6tech3').innerHTML = YouTube.technologies.tech3;
+}
+
+function openModal() {
+  const deskModal = document.getElementById('modalDesktopID');
+  const mobileModal = document.getElementById('modalMobileID');
+
+  document.getElementById('mobileProject').innerHTML = 'Wikipedia';
+  document.getElementById('desktopProject').innerHTML = 'Wikipedia';
+
+  document.getElementById('bigPicMobile').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicMobile1').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicMobile2').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicMobile3').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicMobile4').src = 'resources/Rectangle 21.png';
+
+  document.getElementById('bigPicDesktop').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicDesktop1').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicDesktop2').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicDesktop3').src = 'resources/Rectangle 21.png';
+  document.getElementById('smallPicDesktop4').src = 'resources/Rectangle 21.png';
+
+  if (window.screen.width < 768) {
+    mobileModal.classList.toggle('dispNone');
+  } else {
+    deskModal.classList.toggle('dispNone');
+  }
+}
```

Comment:
- [ ] Having multiple `openModal()` functions like this that basically do the same thing goes against the [DRY](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) principles of programming and I can see you can reuse most of the code in the function.
- Hint: You can create one `openModal(project)` function, the parameter `project` refers to the project object that will get passed depending on which project is clicked and you can then use that object to fill in the rest of the information inside the function. This might take some time so please think it through and I'm sure you'll appreciate how much the code base gets reduced at the end. 💪